data/json/recipes/electronics/parts.json

```diff
@@ -119,6 +119,22 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "scrap", 6 ] ], [ [ "cable", 4 ] ], [ [ "plut_cell", 20 ] ] ]
   },
+  {
+    "result": "flux_comp_gen",
+    "type": "recipe",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "skills_required": [ "fabrication", 4 ],
+    "time": "60 m",
+    "decomp_learn": 4,
```

Comment:
Unless you intend for this to be a reversible recipe (and given materials used, it likely can't be such), I think "decomp_learn" is not needed.

Reply:
Removing now.